CMake configuration error on dev branch

At the following latest commit I just pulled

```
commit 40eee124f42e412311ca8a1700072f276a9d96f3 (HEAD -> dev, origin/dev)
Merge: 514ddccd b5cbbdfc
Author: Barry Fowler <barfowl@gmail.com>
Date:   Fri Sep 9 18:37:40 2022 -0700

    Merge pull request #1262 from barfowl/bfr_evaluate_default
    
    Address failures in default execution of Bfr regression test
```

I am seeing these errors

```
CMake Error at CMakeLists.txt:715 (add_dependencies):
  The dependency target "DEPENDS" of target "public_headers" does not exist.
Call Stack (most recent call first):
  opensubdiv/sdc/CMakeLists.txt:65 (osd_add_doxy_headers)
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -712,7 +712,7 @@ macro(osd_add_doxy_headers headers)
             list(APPEND headerfiles ${targetname} )
         endforeach()
 
-        add_dependencies( public_headers DEPENDS ${headerfiles} )
+        add_dependencies( public_headers ${headerfiles} )
     endif()
 endmacro()
 

